spec/lib/active_campaign/api/fields_spec.rb

```diff
@@ -0,0 +1,129 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe ActiveCampaign::API::Fields, :vcr do
+  let(:client) { ActiveCampaign.client }
+
+  describe '#create_field', :with_text_field_params do
+    subject(:response) { client.create_field(field_params) }
+
+    after do
+      client.delete_field(response.dig(:field, :id))
+    end
+
+    it 'returns a field hash' do
+      expect(response).to include_json(field: expected_field_response)
+    end
+  end
+
+  describe '#show_field', :with_existing_text_field do
+    subject(:response) { client.show_field(field_id) }
+
+    it 'returns a field hash' do
+      expect(response).to include_json(field: expected_field_response)
+    end
+  end
+
+  describe '#show_fields', :with_existing_text_field do
+    subject(:response) { client.show_fields }
+
+    it 'returns a field hash' do
+      expect(response).to include_json(fields: [expected_field_response])
+    end
+  end
+
+  describe '#update_field', :with_existing_text_field do
+    subject(:response) { client.update_field(field_id, update_params) }
+
+    let(:field_title) { 'Custom text - updated' }
```

Comment:
Actually, this might not be testing what you think. Since you are overriding the fields from the original creation the fields will be created with these values and not updated.